pilot/pkg/config/kube/gateway/gateway_ca_controller.go

```diff
@@ -78,18 +83,42 @@ func NewGatewayCAController(kubeClient kube.Client, caBundleWatcher *keycertbund
 	c.gateways.AddEventHandler(controllers.FilteredObjectSpecHandler(c.queue.AddObject, func(o controllers.Object) bool {
 		return o.GetLabels()[label.IoIstioRev.Name] == revision
 	}))
+
+	if features.EnableCACRL {
+		// set filtered client for crl configmap
+		c.crlConfigmaps = kclient.NewFiltered[*v1.ConfigMap](kubeClient, kclient.Filter{
+			FieldSelector: "metadata.name=" + CRLNamespaceConfigMap,
+			ObjectFilter:  kubeClient.ObjectFilter(),
+		})
+
+		// register crl configmap event handler
+		c.crlConfigmaps.AddEventHandler(controllers.ObjectHandler(c.queue.AddObject))
```

Comment:
No need to skip system NSs as upstream does? - https://github.com/nilekhc/istio/blob/6878bb1856a01e4bc9f3bcda9e5d04b75a56ee60/pilot/pkg/serviceregistry/kube/controller/namespacecontroller.go#L117